### DISCUSSION TITLE
Runtime not supported

### DISCUSSION THREAD

**Discussion by Craigoman79** [QuestPDF/QuestPDF#879] [Q&A] [answered]
My API works fine locally but after deploying it to my AzureAppService the endpoint fails. Below referencing the logs on the Azure side.

WAExtractApi.Services.ExportService+ExportException: Error in exporting message PDF:
 ---> System.TypeInitializationException: The type initializer for 'QuestPDF.Settings' threw an exception.
 ---> QuestPDF.Drawing.Exceptions.InitializationException: The QuestPDF library has encountered an issue while loading one of its dependencies.

Your runtime is currently not supported by QuestPDF. Currently supported runtimes are: win-x64, linux-x64, linux-arm64, linux-musl-x64, osx-x64, osx-arm64.

Please consider setting the 'Platform target' property to 'x64' in your project settings.
 ---> System.DllNotFoundException: Unable to load DLL 'QuestPdfSkia' or one of its dependencies: The specified module could not be found. (0x8007007E)
   at QuestPDF.Skia.SkNativeDependencyCompatibilityChecker.API.check_compatibility_by_calculating_sum(Int32 a, Int32 b)
   at QuestPDF.Skia.SkNativeDependencyCompatibilityChecker.CheckIfExceptionIsThrownWhenLoadingNativeDependencies()
   --- End of inner exception stack trace ---
   at QuestPDF.Skia.SkNativeDependencyCompatibilityChecker.Test()
   at QuestPDF.Helpers.NativeDependencyCompatibilityChecker.Test()
   at QuestPDF.Settings..cctor()
   --- End of inner exception stack trace ---
   at QuestPDF.Settings.set_License(Nullable`1 value)

**Comment by Lure5134** [comment, marked as answer]
I think this should help:
https://github.com/QuestPDF/QuestPDF/discussions/622#discussioncomment-9305833

**Reply by Craigoman79** [reply]
Perfect works now. Am I correct in assuming I need to upgrade the costing tier to enable this?

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
